pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: A new Flutter project.
 
 # The following line prevents the package from being accidentally published to
 # pub.dev using `flutter pub publish`. This is preferred for private packages.
-publish_to: 'none' # Remove this line if you wish to publish to pub.dev
+publish_to: "none" # Remove this line if you wish to publish to pub.dev
```

Comment:
Please revert to single quotes. 
Single quotes are preferred in Flutter. I'll take some time to refactor the other quotes on the codebase

ref: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-single-quotes-for-strings

Reply:
I think the prettier workflow updated this... But I will revert

Reply:
Resolved this in the latest push